src/boot/efi/boot.c

```diff
@@ -1035,6 +1085,16 @@ static VOID config_defaults_load_from_file(Config *config, CHAR8 *content) {
                         continue;
                 }
 
+                if (strcmpa((CHAR8 *)"auto-enroll", key) == 0) {
```

Comment:
This should follow the other boolean parsing in style now. They show error messages now and there's no need for the intermediate bool.